### PR TITLE
organization-menu-list-item-cy-217

### PR DESCRIPTION
**Pull request by steiningermona**

### CODE REVIEW

**Review by diegomart2000 — CHANGES_REQUESTED** [review]
@steiningermona please rebase from dev.

File: src/components/badge/Badge.tsx [hunk 1]
```diff
@@ -1,40 +1,59 @@
...
+export interface IBadgeProps {
   value: string;
   position: EBadgePosition;
   disabled?: boolean;
+  type: IBadgeType;
+  size?: IBadgeSizeType;
 }
 
-export const Badge: React.FC<IBadge> = ({ ...props }) => {
+export const Badge: React.FC<IBadgeProps> = ({ children, ...props }) => {
```
Comment (diegomart2000) [inline src/components/badge/Badge.tsx]: Prefer destructuring and named imports for react. Also badge is not working properly. Test here https://league-pxa6boiva-caisy.vercel.app/?path=/story/components-badge--position-left&args=type:color;value:11

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

File: src/components/organization-menu-list-item/OrganizationMenuListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  itemSize?: IListItemSize,
+}
+
+const SFlexListItem = styled(SFlex)`
+  width: 90%;
+  gap: 0.75rem;
+  height: 100%;
+`;
+
+export const OrganizationMenuListItem: React.FC<IOrganizationMenuListItemProps> = ({ ...props }) => {
```
Comment (diegomart2000) [inline src/components/organization-menu-list-item/OrganizationMenuListItem.tsx]: Prefer destructuring and named imports. Only pass down used properties. Make sure component won't break on larger pieces of content.

https://league-pxa6boiva-caisy.vercel.app/?path=/story/components-organizationmenulistitem--default&args=title:Label+kkkk+dsdf+dsfsdfw+wer+wer+werwe+rwer+we+rwer;label:Label+kkkk+dsdf+dsfsdfw+wer+wer+werwe+rwer+we+rwer;size:small

File: src/components/preview-flag-round/PreviewFlagRound.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React, { FC } from "react";
+import Flag from "../flag";
+import { IFlag } from "../flag/IFlag";
+
+const PreviewFlagRound: FC<IFlag> = ({ countryCode }) => {
```
Comment (diegomart2000) [inline src/components/preview-flag-round/PreviewFlagRound.tsx]: This component is repeated and not actually needed, let's discuss on why the original is not being used.

File: src/components/preview-icon/PreviewIcon.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    size: {
+      name: "size",
+      description: "Size of the preview icon",
+      control: { type: "number" },
+      type: "number"
+    },
+  },
+};
+
+function PreviewIconDemo(args) {
```
Comment (diegomart2000) [inline src/components/preview-icon/PreviewIcon.stories.tsx]: This component is repeated and not actually needed, let's discuss on why the original is not being used.

File: src/components/preview-icon/PreviewIcon.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { SPreviewIcon } from "./styles/SPreviewIcon";
+
+export interface IPreviewIcon {
+  icon?: React.ReactNode;
+  size: number;
+}
+
+export const PreviewIcon: React.FC<IPreviewIcon> = ({ icon, ...props }) => {
```
Comment (diegomart2000) [inline src/components/preview-icon/PreviewIcon.tsx]: This component is repeated and not actually needed, let's discuss on why the original is not being used.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

**Comment by steiningermona** [comment]
**from figma**
image
**from storybook**
- small/ medium
image
- large with badge
image

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**